.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [main]
```

Comment:
think this should be `master` in this repo

Reply:
Yep!